language/tools/move-cli/tests/sandbox_tests/module_publish_view/args.exp

```diff
@@ -4,8 +4,7 @@ Publishing a new module 00000000000000000000000000000042::Module (wrote 120 byte
 Wrote 120 bytes of module ID's and code
 Command `sandbox view storage/0x00000000000000000000000000000042/modules/Module.mv`:
 // Move bytecode v6
-module 42.Module {
-struct S {
+module 42.Module {struct S {
```

Comment:
This seems not right.

Reply:
Good call, forgot to update the output here